### ISSUE TITLE
[Bug]: Accessibility Issue with DataGrid Component

### ISSUE THREAD

**Issue by SamuelMorinFr** [microsoft/fluentui#27276]
### Library
Fluent UI React v9 (@fluentui/react-components)
### Issue
The `DataGrid` component violates the `aria-required-children` [accessibility rule](https://accessibilityinsights.io/info-examples/web/aria-required-children), because the parent element has a `grid` role but it contains 2 `<i>` child elements, at first and last position, with `role=none`.
#### Target application
[Components / DataGrid - Default ⋅ Storybook](https://react.fluentui.dev/?path=/docs/components-datagrid--default)
#### Element path
#storybook-preview-iframe;.fui-FluentProvider58 > div > .fui-DataGrid.fui-Table.___1kbqy50
#### Snippet
`<div role="grid" data-tabster="{&quot;mover&quot;:{&quot;cyclic&quot;:false,&quot;direction&quot;:3}}" class="fui-DataGrid fui-Table ___1kbqy50 fgkb47j fhovq9v ftgm304">`
#### How to fix
    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute
#### Environment
Microsoft Edge version 111.0.1661.44
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![image](https://user-images.githubusercontent.com/100727705/226654313-fd4f46f1-f153-4447-bef0-191894a62cf1.png)

**Comment by behowell** [comment]
Similar to #27232, the `<i>` elements that it's flagging as invalid children have `aria-hidden="true"`. Seems like it might be a false positive?

**Comment by y226xie** [comment]
This issue is also noticed with the V9 `Table` component, when `Table` has `tabbable: true` property is added to the `useArrowNavigationGroup`. 

Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (.fui-Table)
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Element path: .fui-Table

Snippet: <div role="grid" aria-rowcount="26" data-tabster="{&quot;mover&quot;:{&quot;cyclic&quot;:false,&quot;direction&quot;:1,&quot;tabbable&quot;:true}}" class="fui-Table ___1kbqy50_18rlg51 fgkb47j fhovq9v ftgm304">

Related paths: 
  i:nth-child(1)
  i:nth-child(4)

How to fix: 
Fix any of the following:
  Element has children which are not allowed (see related nodes)
  Element has no aria-busy="true" attribute

Environment: Chrome version 113.0.0.0
